lib/flipper/api/v1/actions/percentage_of_actors_gate.rb

```diff
@@ -0,0 +1,57 @@
+require 'flipper/api/action'
+require 'flipper/api/v1/decorators/feature'
+
+module Flipper
+  module Api
+    module V1
+      module Actions
+        class PercentageOfActorsGate < Api::Action
+          route %r{api/v1/features/[^/]*/percentage_of_actors/?\Z}
+
+          def post
+            ensure_valid_enable_params
+            feature = flipper[feature_name]
+            feature.enable_percentage_of_actors(percentage)
+            decorated_feature = Decorators::Feature.new(feature)
+            json_response(decorated_feature.as_json, 200)
+          end
+
+          def delete
+            ensure_valid_disable_params
+            feature = flipper[feature_name]
+            feature.disable_percentage_of_actors
+            decorated_feature = Decorators::Feature.new(feature)
+            json_response(decorated_feature.as_json, 200)
+          end
+
+          private
+
+          def ensure_valid_enable_params
+            json_error_response(:feature_not_found) unless feature_names.include?(feature_name)
+            json_error_response(:percentage_invalid) unless number_less_than_or_equal_100?(params['percentage'])
+          end
+
+          def ensure_valid_disable_params
+            json_error_response(:feature_not_found) unless feature_names.include?(feature_name)
+          end
+
+          def feature_name
+            @feature_name ||= Rack::Utils.unescape(path_parts[-2])
+          end
+
+          def percentage
+            @percentage ||= params['percentage'].to_i
+          end
+
+          def feature_names
+            @feature_names ||= flipper.adapter.features
+          end
+
+          def number_less_than_or_equal_100?(val)
+            !!/^(100|[1-9][0-9]?)$/.match(val)
```

Comment:
Technically you can enable for 0 percent using flipper and the flipper-ui. Do you feel we should support the same semantics for the API or just point people to disable?